[W16-1] Hallper

pixelky opened the pull request.

bqnguyen94 commented:
@EatOrBeEaten Let me know when the PR is ready for review :)

bqnguyen94 commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,61 @@
-AddressBook - Level 4 was developed by the https://se-edu.github.io/docs/Team.html[se-edu] team. +
-_{The dummy content given below serves as a placeholder to be used by future forks of the project.}_ +
+Hallper was developed by https://cs2103-ay1819s1-w16-1.github.io/main/[TEAM W16-1].  +
++
```
This "+" sign tho

bqnguyen94 commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,61 @@
+=== ERIC YANG JIA WEI
+image::ericyjw.jpeg[width="150", align="left"]
+{empty}[https://github.com/ericyjw[github]] [<<ericjyw#, portfolio>>] +
+_{#To add in portfolio#}_ +
```
Pls remove this line

bqnguyen94 commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,61 @@
 
-Role: Project Advisor
+Role: Developer +
+Responsibilities: [To be filled up]
```
And update the responsibilities. You can put something you have in mind now and update it anytime in the future :)

pixelky commented:
@bqnguyen94 PR is ready for review

bqnguyen94 reviewed (commented):
Looks good overall 👍 just a few missing pieces. Also, rmb not to close this PR as it will be used to track your project's progresses in subsequent weeks :)

pixelky commented:
@bqnguyen94 Ready for 1.1 review.

bqnguyen94 reviewed (commented):
Some notes: 
* Do change the description on the top of your repo's Github. Right now it is saying `addressbook-level4 https://se-edu.github.io/addressbook-level4`
* In netlify-generated landing page, you can modify (or remove) the top navigation bar
* Update (or remove) the badges on top of `README` to point to your repo's status
* `AboutUs`: some of the profile pictures the faces abit small
* `UserGuide`: add `coming in v2.0` to features that hasn't been implemented yet

I see that you guys have made quite a progress over the past weeks, good job! 👍

GilgameshTC commented:
@bqnguyen94 PR is ready for review

pixelky commented:
@bqnguyen94 Ready for v1.2 review

bqnguyen94 reviewed (commented):
* I see that you have updated your documents as per requested 👍 
* You have also customized the Developer Guide after your application, well done 👍 if possible, do update the images and diagrams as well; and there are places throughout the document that the application's name is still `AddressBook`. Also, for the introduction section, perhaps you can give a brief guide on how to use the DG, e.g. what each section contains, where should the reader start from if they are getting started, or if they have already finished setting up and want to dive into the details, etc.

pixelky commented:
@bqnguyen94 Netlify seems to have failed due to Bad Gateway.

ericyjw committed:
Add UpdateCommandParserTest

Commit:
Added EmailModelTest

Commit:
Edited getExistingEmails to return unmodifiableSet

Commit:
Added test case of EmailModelTest

ericyjw committed:
Add BudgetCommandParserTest

ericyjw committed:
Add DeleteTransactionCommandParserTest

Commit:
Added test cases to EmailModelTest

Commit:
Added equals method to EmailModel

Commit:
Edited equals method in ModelManager and equals test in ModelManagerTest

Commit:
Fixed codacy issue

ericyjw committed:
Add Test cases for Addressbook parser

ericyjw committed:
Add DeleteTransactionCommandTest

ericyjw committed:
Add TransactionMathTest

kengwoon committed:
Updated UserGuide

Commit:
Added test data for EmailDirStorageTest

Commit:
Added EmailDirStorageTest

kengwoon committed:
Updated PPP

ericyjw committed:
Add BudgetBrowserPanelTest

ericyjw committed:
Add CcaCardTest

Commit:
Added unit tests to EmailDirStorageTest

Commit:
Removed test data for EmailDirStorageTest

Commit:
Fixed checkstyle errors

Commit:
Added EmailUtil

ericyjw committed:
Update CheckStyle

ericyjw committed:
Merge branch 'develop' of https://github.com/CS2103-AY1819S1-W16-1/main

# Conflicts:
#	docs/UserGuide.adoc

ericyjw committed:
Improve Code Quality

ericyjw committed:
Update CheckStyle

ericyjw committed:
Update CheckStyle

javenseow committed:
Updated User Guide and Project Portfolio Page

ericyjw committed:
Fix #143

GilgameshTC committed:
Implemented OS check

If OS is not Windows, calendar content parsed will have its
line separator replaced.

GilgameshTC committed:
Modified hard coded line separator replacement in CalendarPanel

javenseow committed:
updated ProfilePictureTest.java and created ImageCommandParserTest.java

GilgameshTC committed:
Changed calendar path to "calendars"

GilgameshTC committed:
Made checkstyle changes

GilgameshTC committed:
Merge pull request #164 from GilgameshTC/calendar-view-fix

Resolve Calendar UI on Mac OS bug

GilgameshTC committed:
Merge upstream develop

GilgameshTC committed:
Merge branch 'develop' of https://github.com/CS2103-AY1819S1-W16-1/main into calendar-testcases

GilgameshTC committed:
Remove view_calendar not working on Mac OS msg in UG

GilgameshTC committed:
Implemented CreateCalendarCommandParserTest

bqnguyen94 reviewed (commented):
Some comments on your developer guide:
* Please update the diagrams ASAP. Remove references to `AddressBook`, and add your new features' behaviors.
* Some sections are incomplete (4.7, Appendix H.3), update or omit them.
* Replace all other references to `AddressBook` in the text.

Commit:
Merge branch 'develop' into email

GilgameshTC committed:
Raised ToggleBrowserPlaceholderEvent in relevant methods

GilgameshTC committed:
Raised ToggleBrowserPlaceholderEvent in relevant methods

javenseow committed:
updated browser panel and created SearchCommandSystemTest.java

javenseow committed:
Merge pull request #33 from CS2103-AY1819S1-W16-1/develop

Develop

Commit:
Merge branch 'develop' into email

GilgameshTC committed:
Implemented CalendarPanelHandle

Added a default Calendar to be loaded in CalendarPanel

ericyjw committed:
Update Developer Guide

GilgameshTC committed:
Implemented AddAllDayEventCommandTest

GilgameshTC committed:
Made checkstyle changes

GilgameshTC committed:
Implemented AddEventCommandTest

GilgameshTC committed:
Implemented AddAllDayEventCommandParserTest

GilgameshTC committed:
Made Checkstyle changes

GilgameshTC committed:
Implemented AddEventCommandParserTest

GilgameshTC committed:
Implemented DeleteEventCommandTest

GilgameshTC committed:
Implemented ViewCalendarCommandTest

GilgameshTC committed:
Implemented ViewCalendarCommandParserTest

GilgameshTC committed:
Implemented DeleteEventCommandParserTest

GilgameshTC committed:
Updated application to handle removal of calendars in local storage.

GilgameshTC committed:
Updated CalendarModelTest

GilgameshTC committed:
Updated StorageManagerTest to handle IcsCalendarStorage

GilgameshTC committed:
Implemented ViewCalendarCommandSystemTest

ericyjw committed:
Update Developer Guide with Sequence Diagram &
Fix #153,#146, #138

ericyjw committed:
Update CheckStyle

kengwoon committed:
ClearCommand changes

ericyjw committed:
Merge branch 'develop' of https://github.com/CS2103-AY1819S1-W16-1/main

# Conflicts:
#	docs/UserGuide.adoc
#	src/main/java/seedu/address/model/UserPrefs.java
#	src/main/java/seedu/address/model/person/ProfilePicture.java
#	src/main/resources/ProfileWindow.html
#	src/test/java/seedu/address/model/person/ProfilePictureTest.java

kengwoon committed:
EraseCommand changes

ericyjw committed:
Update Test Case

ericyjw committed:
Update CheckStyle

javenseow committed:
Updated Documentations and included tests

javenseow committed:
Corrected checkstyle errors

javenseow committed:
Corrected checkstyle errors

ericyjw committed:
Update PPP, UserGuide, Developer Guide

ericyjw committed:
Update CheckStyle

ericyjw committed:
Removed condition

kengwoon committed:
Updated ClearCommandSystemTest

kengwoon committed:
Checkstyle changes

ericyjw committed:
Update checkstyle

ericyjw committed:
Change xsl file path in user pref

ericyjw committed:
Remove xsl

kengwoon committed:
Added EraseCommandSystemTest

ericyjw committed:
Move readXsl to Model

kengwoon committed:
Updated DG with sequence diagrams

ericyjw committed:
Update BudgetTestFile

ericyjw committed:
Update CheckStyle

ericyjw committed:
test

ericyjw committed:
test

ericyjw committed:
test

javenseow committed:
included EOF for ccabook.xsl

javenseow committed:
edited length of Search Command Test

javenseow committed:
Corrected checkstyle errors in Search Command System Test

javenseow committed:
correct Search Command System Test error

javenseow committed:
corrected Search Command System Test checkstyle error

ericyjw committed:
test

kengwoon committed:
Added activity diagrams to DG

javenseow committed:
included sequence diagram for ImageCommand to developer guide

kengwoon committed:
Updated ClearCommand

kengwoon committed:
Updated PPP

ericyjw committed:
test

kengwoon committed:
Checkstyle changes

ericyjw committed:
update checkstyle

ericyjw committed:
update budget test case

javenseow committed:
Updated documentation and included ImageCommandTest.java

javenseow committed:
included more tests in Image Command Test

kengwoon committed:
Updated PPP and AboutUs

kengwoon committed:
Merge pull request #134 from kengwoon/v1.4

add more Import types, added system tests and updated UGDG and PPP

javenseow committed:
updated Image Command Test

GilgameshTC committed:
Merge pull request #119 from GilgameshTC/calendar-testcases

Implement Test Cases for Calendar resolves #58

javenseow committed:
Merge branch 'develop' into search

javenseow committed:
corrected checkstyle error

javenseow committed:
Merge branch 'search' of https://github.com/javenseow/main into search

ericyjw committed:
Merge branch 'develop' of https://github.com/CS2103-AY1819S1-W16-1/main

# Conflicts:
#	docs/AboutUs.adoc
#	docs/UserGuide.adoc
#	docs/ccabook.xsl
#	src/main/java/seedu/address/storage/Storage.java
#	src/main/java/seedu/address/storage/StorageManager.java
#	src/test/java/seedu/address/logic/commands/CommandTestUtil.java

javenseow committed:
updated Image Command Test

ericyjw committed:
Update CheckStyle

javenseow committed:
corrected filepath in image command test

ericyjw committed:
Merge pull request #162 from ericyjw/master

Test Cases, Project Portfolio, Issues Update

javenseow committed:
renamed Valid.jpg for Image Command Test

javenseow committed:
Merge branch 'develop' into search

javenseow committed:
Merge pull request #166 from javenseow/search

Updated documentation and created test cases

javenseow committed:
Merge pull request #34 from CS2103-AY1819S1-W16-1/develop

Merge pull request #166 from javenseow/search

javenseow committed:
updated README.adoc to exclude Apache.org library

ericyjw committed:
Update UserGuide

ericyjw committed:
Update PPP

ericyjw committed:
Merge branch 'develop' of https://github.com/CS2103-AY1819S1-W16-1/main

# Conflicts:
#	docs/UserGuide.adoc

Commit:
Moved raising EmailViewEvent to ModelManager

Commit:
Added saveComposedEmailWithoutDisplay to ModelStub in AddCommandTest

Commit:
Added DeleteEmailCommandSystemTest

ericyjw committed:
Update Developer Guide

Commit:
Merge branch 'develop' into email

Commit:
Added method to modelstub in CreateCcaCommandTest

Commit:
Merge pull request #160 from EatOrBeEaten/email

Tests and documentation for Email features

javenseow committed:
updated documentation

ericyjw committed:
Merge branch 'develop' of https://github.com/CS2103-AY1819S1-W16-1/main

# Conflicts:
#	docs/UserGuide.adoc

ericyjw committed:
Merge pull request #168 from ericyjw/master

Update documentations

javenseow committed:
Updated documentation and tagged Git username

javenseow committed:
Deleted product survery from developer guide

Commit:
Changed README email description

Commit:
Changed figure numbers for email and list

javenseow committed:
Merge pull request #167 from javenseow/search

Documentation update

Commit:
Added delete_email and list_emails images

Commit:
Merge branch 'develop' into email

kengwoon committed:
Merge pull request #2 from CS2103-AY1819S1-W16-1/develop

Develop

kengwoon committed:
Added description for figures in DG

javenseow committed:
included code contributed to javenseow.adoc

javenseow committed:
Merge pull request #172 from javenseow/search

Updated PPP

kengwoon committed:
Updated Ui image

kengwoon committed:
Merge pull request #170 from kengwoon/v1.4

Documentation changes

ericyjw committed:
Update PPP with Reposense

Commit:
Removed unused fields in CommandTestUtil

javenseow committed:
Merge pull request #35 from CS2103-AY1819S1-W16-1/develop

Develop

Commit:
adjusted captions

ericyjw committed:
Add DeleteCcaCommandSystemTest

ericyjw committed:
Add DeleteCcaCommandSystemTest

ericyjw committed:
Merge pull request #173 from ericyjw/master

System test case

Commit:
removed site-seedu from build.gradle

Commit:
Edited 'updating documentation' in DG

Commit:
Added sequence diagram for email feature

GilgameshTC committed:
Added Activity and Sequence Diagrams of Calendar Features to DG

Commit:
Edited eatorbeeaten.adoc

Commit:
Edited main features in eatorbeeaten.adoc

GilgameshTC committed:
Update Travis Project Portfolio

Commit:
Merge pull request #169 from EatOrBeEaten/email

Update email documentation

Commit:
Merge pull request #174 from GilgameshTC/calendar-guide

Updated Calendar Feature DG

javenseow committed:
Merge pull request #36 from CS2103-AY1819S1-W16-1/develop

Develop

javenseow committed:
Updated responsibilities in AboutUs.adoc

Commit:
Merge pull request #175 from CS2103-AY1819S1-W16-1/develop

Update Master branch from Develop

javenseow committed:
Merge pull request #176 from javenseow/search

Updated AboutUs.adoc on responsibilities

Commit:
Added EmailActivityDiagram to DG

Commit:
Merge branch 'master' into email

GilgameshTC committed:
Added numbering to headers in Travis Project Portfolio

GilgameshTC committed:
Added JFxtras into README

GilgameshTC committed:
Updated Documentation contribution in PPP

ericyjw committed:
Update DeveloperGuide and PPP

GilgameshTC committed:
Updated Diagrams of Calendar Feature in UG

kengwoon committed:
PPP changes

Commit:
Renumbered all figures and tables in DG. Added test cases for compose email.

Commit:
Edited UG intro

Commit:
Fixed typo and chekstyle error

kengwoon committed:
UG changes

GilgameshTC committed:
Updated StorageClassDiagram in DG

kengwoon committed:
Removed EraseCommand UGDG sections

kengwoon committed:
Merge pull request #179 from kengwoon/v1.4

PPP changes

kengwoon committed:
Shorten PPP

ericyjw committed:
Update PPP

Commit:
Merge pull request #180 from EatOrBeEaten/email

Update documentation

GilgameshTC committed:
Updated Travis PPP

kengwoon committed:
Merge pull request #181 from kengwoon/v1.4

Documentation updates

ericyjw committed:
Merge pull request #182 from ericyjw/master

Update Documentation

kengwoon committed:
DG Appendix G updates

kengwoon committed:
Merge branch 'master' into v1.4

GilgameshTC committed:
Merge pull request #178 from GilgameshTC/calendar-guide

Update README & UG

kengwoon committed:
Added more test cases to Appendix G in DG

kengwoon committed:
Merge pull request #183 from kengwoon/v1.4

DG Appendix G updates

kengwoon committed:
PPP intro changes

kengwoon committed:
Merge pull request #184 from kengwoon/v1.4

PPP intro changes

kengwoon committed:
Modified BudgetBookExample image

kengwoon committed:
Merge pull request #185 from kengwoon/v1.4

Modified BudgetBookExample image

javenseow committed:
Documentation update

javenseow committed:
Merge branch 'develop' into search

javenseow committed:
Merge pull request #186 from javenseow/search

Documentation update

javenseow committed:
Merge branch 'master' into develop

ericyjw committed:
Update User Guide

GilgameshTC committed:
Made changes to Travis PPP

GilgameshTC committed:
Merge remote-tracking branch 'upstream/master' into upstream-master

javenseow committed:
Merge pull request #177 from CS2103-AY1819S1-W16-1/develop

Updated documentation

javenseow committed:
Merge pull request #37 from CS2103-AY1819S1-W16-1/master

Update local

javenseow committed:
Merge pull request #38 from javenseow/master

Update

ericyjw committed:
Update Developer Guide Appendix H

javenseow committed:
updated documentation

javenseow committed:
adjusted diagram sizes in developer guide

javenseow committed:
added EOF in developer guide

javenseow committed:
correct checkstyle error in developer guide

javenseow committed:
correct developer guide

ericyjw committed:
Update Developer Guide Use Case

GilgameshTC committed:
Added Calendar Feature to Manual Testing Appendix G in DG

GilgameshTC committed:
Merge remote-tracking branch 'upstream/master' into upstream-master

# Conflicts:
#	docs/UserGuide.adoc

GilgameshTC committed:
Made grammar changes to Travis PPP

ericyjw committed:
Merge pull request #187 from javenseow/search

Updated documentation

ericyjw committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W16-1/main

# Conflicts:
#	docs/DeveloperGuide.adoc

ericyjw committed:
Add newline at EOF for Developer Guide

javenseow committed:
Updated model class diagram

ericyjw committed:
Fix Test Cases

ericyjw committed:
Fix Delete Entry Message

javenseow committed:
Merge pull request #189 from javenseow/search

Updated model class diagram

Commit:
removed calendar folder

ericyjw committed:
Update Appendix G

Commit:
Merge branch 'master' into email

GilgameshTC committed:
Made grammar changes to CalendarCommandActivityDiagram

GilgameshTC committed:
Merge remote-tracking branch 'upstream/master' into upstream-master

Commit:
Added Manual testing for email feature

ericyjw committed:
Update PPP

ericyjw committed:
Update PPP

Commit:
Updated EmailActivityDiagram

Commit:
Edited EmailActivityDiagram

Commit:
Edited size of EmailActivityDiagram

Commit:
Edited overview and summary in eatorbeeaten.adoc

ericyjw committed:
Update PPP

Commit:
Added condition for add command in UG

Commit:
Merge pull request #188 from ericyjw/master

Update documentation

Commit:
Merge pull request #190 from EatOrBeEaten/email

Update documentations

Commit:
Edited casing for G22-24

Commit:
Merge pull request #191 from EatOrBeEaten/email

Edited casing for G22-24

Commit:
Edited user stories in DG

ericyjw committed:
Updated budget command

Commit:
fixed grammatical error in DG

ericyjw committed:
Add new line at ccabook.xsl EOF

Commit:
Merge pull request #193 from EatOrBeEaten/email

Update documentation

Commit:
Merge pull request #192 from ericyjw/master

Fix Budget Command

ericyjw committed:
Update all documents

Commit:
Merge pull request #194 from ericyjw/master

Update all documents

pixelky commented:
v1.4 done
